pkgs/servers/ttyd/default.nix

```diff
@@ -1,30 +1,62 @@
 { lib, stdenv, fetchFromGitHub
 , pkg-config, cmake, xxd
 , openssl, libwebsockets, json_c, libuv, zlib
+, python3, nodejs-slim, yarn, fixup_yarn_lock, yarn2nix
+, callPackage, runCommand
 }:
 
 with builtins;
 
 stdenv.mkDerivation rec {
   pname = "ttyd";
-  version = "1.6.1";
+  version = "1.6.3";
   src = fetchFromGitHub {
     owner = "tsl0922";
     repo = pname;
-    rev = "refs/tags/${version}";
-    sha256 = "1ifgw93g8jaaa6fgfqjnn83n5ccr6l72ynwwwa97hfwjk90r14fg";
+    rev = version;
+    sha256 = "0yks0dz0prajvq6aksznih6ggxf337c1amhkj5flkn7cv3vrvd8j";
   };
 
-  nativeBuildInputs = [ pkg-config cmake xxd ];
+  nativeBuildInputs = [ pkg-config cmake xxd yarn nodejs-slim python3 ];
   buildInputs = [ openssl libwebsockets json_c libuv zlib ];
 
+  # The file src/html.h in the ttyd repository is generated from the frontend
+  # source in the html/ subdirectory. Instead of using this pre-built file, we
+  # generate it ourselves. We do this not only to keep everything honest, but
+  # also to facilitate easy overriding/patching of this package. Changes to the
+  # html/ subdirectory wouldn't have any effect without regenerating src/html.h.
+  preBuild =
+    let
+      yarnNixFile = runCommand "yarn.nix" {}
+        "${yarn2nix}/bin/yarn2nix --lockfile ${src}/html/yarn.lock --no-patch --builtin-fetchgit > $out";
+      yarnNix = callPackage yarnNixFile {};
```

Comment:
You can't do this in nixpkgs and so ofborg fails. So I think you need to add the lock file.